tests/e2e/ilab_rhoai_pipeline_runs_test.go

```diff
@@ -0,0 +1,133 @@
+)
+
+type PipelineParams struct {
+	FinalEvalBatchSize              string  `yaml:"final_eval_batch_size" json:"final_eval_batch_size"`
+	FinalEvalFewShots               int     `yaml:"final_eval_few_shots" json:"final_eval_few_shots"`
+	FinalEvalMaxWorkers             string  `yaml:"final_eval_max_workers" json:"final_eval_max_workers"`
+	FinalEvalMergeSystemUserMessage bool    `yaml:"final_eval_merge_system_user_message" json:"final_eval_merge_system_user_message"`
+	K8SStorageClassName             string  `yaml:"k8s_storage_class_name" json:"k8s_storage_class_name"`
+	MTBenchMaxWorkers               string  `yaml:"mt_bench_max_workers" json:"mt_bench_max_workers"`
+	MTBenchMergeSystemUserMessage   bool    `yaml:"mt_bench_merge_system_user_message" json:"mt_bench_merge_system_user_message"`
+	SDGBaseModel                    string  `yaml:"sdg_base_model" json:"sdg_base_model"`
+	SDGMaxBatchLen                  int     `yaml:"sdg_max_batch_len" json:"sdg_max_batch_len"`
+	SDGPipeline                     string  `yaml:"sdg_pipeline" json:"sdg_pipeline"`
+	SDGRepoBranch                   string  `yaml:"sdg_repo_branch" json:"sdg_repo_branch"`
+	SDGRepoPR                       int     `yaml:"sdg_repo_pr" json:"sdg_repo_pr"`
+	SDGRepoURL                      string  `yaml:"sdg_repo_url" json:"sdg_repo_url"`
+	SDGSampleSize                   float64 `yaml:"sdg_sample_size" json:"sdg_sample_size"`
+	SDGScaleFactor                  int     `yaml:"sdg_scale_factor" json:"sdg_scale_factor"`
+	TrainEffectiveBatchSizePhase1   int     `yaml:"train_effective_batch_size_phase_1" json:"train_effective_batch_size_phase_1"`
+	TrainEffectiveBatchSizePhase2   int     `yaml:"train_effective_batch_size_phase_2" json:"train_effective_batch_size_phase_2"`
+	TrainLearningRatePhase1         float64 `yaml:"train_learning_rate_phase_1" json:"train_learning_rate_phase_1"`
+	TrainLearningRatePhase2         float64 `yaml:"train_learning_rate_phase_2" json:"train_learning_rate_phase_2"`
+	TrainMaxBatchLen                int     `yaml:"train_max_batch_len" json:"train_max_batch_len"`
+	TrainNNodes                     int     `yaml:"train_nnodes" json:"train_nnodes"`
+	TrainNProcPerNode               int     `yaml:"train_nproc_per_node" json:"train_nproc_per_node"`
+	TrainNumEpochsPhase1            int     `yaml:"train_num_epochs_phase_1" json:"train_num_epochs_phase_1"`
+	TrainNumEpochsPhase2            int     `yaml:"train_num_epochs_phase_2" json:"train_num_epochs_phase_2"`
+	TrainNumWarmupStepsPhase1       int     `yaml:"train_num_warmup_steps_phase_1" json:"train_num_warmup_steps_phase_1"`
+	TrainNumWarmupStepsPhase2       int     `yaml:"train_num_warmup_steps_phase_2" json:"train_num_warmup_steps_phase_2"`
+	TrainSaveSamples                int     `yaml:"train_save_samples" json:"train_save_samples"`
+	TrainSeed                       int     `yaml:"train_seed" json:"train_seed"`
+}
+
+func loadPipelineParams(filename string) (*PipelineParams, error) {
```

Comment:
can we get some `t.log` statements for some user feedback, I would be interested in at least the following: 

```
Parameter config loaded successfully
Pipeline loaded successfully
Pipeline with name {name} and run id {run_id} started....
Pipeline with name {name} and run id {run_id} finished successfully!
```